src/content/Records.js

```diff
@@ -154,6 +154,33 @@ let p2 = { -y | p1 };`}</CodeBlock>
 
                 </SubSection>
 
+                <SubSection name="Records as named parameters">
+                    <p>
+                        In general, Flix lacks named parameters.
+                        However, to improve usability of certain functions whose parameters might be confusing,
+                        singleton records can be used to clarify parameter order.
```

Comment:
Keep it positive and keep focus on the ball. Try something like (iterate on this):

When a function has multiple parameters that share the same type, it is easy to get confused about the right argument order. For example, what does `String.contains("Hello", "Hello World")` return? What does `String.contains("Hello World", "Hello")`.

A common solution to this problem is to used /named parameters/. Flix supports a form of named parameters building on records. For example, we can write a function `translate` to translate from one language to another as follows: